openapi.yaml

```diff
@@ -0,0 +1,681 @@
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceBinding'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '422':
+          description: Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      summary: deprovision of a service binding
+      tags:
+        - ServiceBindings
+      operationId: serviceBinding.unbinding
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: id of the instance associated with the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: binding_id
+          in: path
+          description: id of the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service associated with the instance for which a binding is being deleted
+          required: true
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan associated with the instance for which a binding is being deleted
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '410':
+          description: Gone
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+
+components:
+  parameters:
+    APIVersion:
+      name: X-Broker-API-Version
+      in: header
+      description: version number of the Service Broker API that the Platform will use
+      required: true
+      schema:
+        type: string
+      default: '2.13'
+
+    OriginatingIdentity:
+      name: X-Broker-API-Originating-Identity
+      in: header
+      description: identity of the user that initiated the request from the Platform
+      schema:
+        type: string
+
+  schemas:
+    Catalog:
+      type: object
+      properties:
+        services:
+          type: array
+          items:
+            $ref: '#/components/schemas/Service'
+
+    Service:
```

Comment:
I believe this and all others need to include the `type: object` attribute, otherwise you're saying they can also be arrays or primitives. At least that's how it was in OpenAPI 2.0.

Reply:
Nice catch!